README.md

```diff
@@ -481,6 +481,47 @@ dsq> SELECT * FROM {} WHERE NAME = 'Kevin';
 (0 rows)
 ```
 
+### Converting numbers in CSV and TSV files
+
+CSV and TSV files do not allow to specify the type of the individual
+values contained in them. All values are treated as strings by default.
+
+This can lead to unexpected results in queries. Consider the following
+example:
+
+```
+$ cat scores.csv
+name,score
+Fritz,90
+Rainer,95
+Fountainer,100
```

Comment:
To demonstrate both integers and floats could you make one of these numbers a float? I.e. with a .0 suffix. Thanks!

Reply:
Updated 👍